lib/std/json.zig

```diff
@@ -110,6 +110,9 @@ pub const WriteStream = @import("json/stringify.zig").WriteStream;
 pub const encodeJsonString = @import("json/stringify.zig").encodeJsonString;
 pub const encodeJsonStringChars = @import("json/stringify.zig").encodeJsonStringChars;
 
+pub const Formatter = @import("json/fmt.zig").Formatter;
```

Comment:
Do we need `Formatter` to be public? Is it because it's a return type?

Reply:
Yeah it's the return type of the new `fmt` function so I figured it's better to make it public just in case?

Reply:
Do you think there is any need to make it public though? What can the potential consumer use it for except implicitly passing into a function accepting a formatter such as `std.debug.print("{}\n", .{std.json.fmt(x)})` or am I missing something?